app/client/styles/main.styl

```diff
@@ -105,10 +109,21 @@ input, textarea
 #header
   width: 100%
   background-color: $color-white
-  box-shadow: 0 3px 5px $color-grey
+  box-shadow: 0 2px 3px $color-grey-7
   position: relative
   z-index: 900
 
+#add-photo-input
+  width: 0.1px
```

Comment:
I think the convention for this kinda thing is absolute position with huge negative top and left values so it could never be on screen. This seems to work but I don't know if there is a good reason for the convention